### PR TITLE
Men outfits

### PR DESCRIPTION
**Pull request by Salam-Dalloul**
### Fixes
fixes #<24>

### Related
related to #<issue_number>

### Summary
upload outfits from database
### Major Changes
[**OPTIONAL for small PRs**] Detailed description of major changes in the PR, ideally with references to relevant commits, issues, line-numbers, etc.

### CODE REVIEW

**Review by walaamedhat — COMMENTED** [review]
just write the number of issue you''re fixed it right here and in the last 2 pull request :+1:

File: public/js/womenFashion.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+function btnClick() {
```
Comment (walaamedhat) [inline public/js/womenFashion.js:1]: you can write this function in one file and use it any where you want

**Review by walaamedhat — DISMISSED** [review]

**Review by muhmushtaha — DISMISSED** [review]
we don't have the outfits table, how you did the route?
plz, test the routes before pushing.

**Review by muhmushtaha — CHANGES_REQUESTED** [review]
we don't have the outfits table, how you did the route?
plz, test the routes before pushing.

**Review by Ghadeer93 — APPROVED** [review]

**Pull request merged**